docs/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.md

```diff
@@ -145,13 +145,13 @@ public class CatalogService : ICatalogService
 }
 ```
 
-The Typed Client (`CatalogService` in the example) is activated by DI (Dependency Injection), meaning that it can accept any registered service in its constructor, in addition to `HttpClient`.
+The Typed Client (`CatalogService` in the example) is activated by DI (Dependency Injection), that means it can accept any registered service in its constructor, in addition to `HttpClient`.
 
-A Typed Client is, effectively, a transient object, meaning that a new instance is created each time one is needed and it will receive a new `HttpClient` instance each time it's constructed. However, the `HttpMessageHandler` objects in the pool are the objects that are reused by multiple `HttpClient` instances.
+A Typed Client is effectively a transient object, that means a new instance is created each time one is needed. And it will receive a new `HttpClient` instance each time it's constructed. However, the `HttpMessageHandler` objects in the pool are the objects that are reused by multiple `HttpClient` instances.
```

Comment:
```suggestion
A Typed Client is effectively a transient object, that means a new instance is created each time one is needed. It receives a new `HttpClient` instance each time it's constructed. However, the `HttpMessageHandler` objects in the pool are the objects that are reused by multiple `HttpClient` instances.
```

No need to start the sentence with "And". Also, use [active voice](https://docs.microsoft.com/en-us/contribute/dotnet/dotnet-voice-tone#use-active-voice).

Reply:
Thank you @IEvangelist for your review and comments. Will incorporate these suggestions.